extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -139,19 +140,20 @@ class QueueProcessor extends EventEmitter {
                     this.logger.info('queue processor is ready to consume ' +
                         'notification entries');
                     this.emit('ready');
+                    return next();
```

Comment:
But in this case, how are we supposed to end the function, if this event is called? Do you mean that the `ready` event will be always called, or, in case of error, the service will crash until a stable state is reached?

Reply:
I mean, if we get the `error` event and `ready` is never called, you will never reach the end of your async.series, as the `next` will never be called.

Reply:
Yes i meant in case of error the service will crash, but fair enough the function has to end if it fails at startup to make things cleaner